app/assets/stylesheets/style.scss

```diff
@@ -490,6 +414,11 @@ label:hover {
 
 .actions .like {
   line-height: 120px;
+  background-color: #83D566;
```

Comment:
Color literals like `#83D566` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#83D566` should be written as `#83d566`